docs/cli-spec.md

```diff
@@ -1,29 +1,27 @@
 ## Polystat usages
 
 ### Notation
-`|` - exclusive or
-`(...)` - must be present
-`[...]` - may not be present
-`<thing>...` -- multiple instances of thing can be accepted.
+The description follows [this guide](https://en.wikipedia.org/wiki/Command-line_interface#:~:text=Command%20description%20syntax%5Bedit%5D).
```

Comment:
This link refers to the relevant section in the article ([link](https://en.wikipedia.org/wiki/Command-line_interface#Command_description_syntax)). Taken from the *Contents* session from the top of the page.